samples/ONNX37_opengpu/onnx_37_app/packages/028663699634-onnx_37_app-1.0/src/runonnx.py

```diff
@@ -159,25 +161,25 @@ def run(self):
                 self.metrics_handler.put_metric(total_process_metric)
                 
                 # Post Process
-                try:
-                    
-                    prediction = self.postprocess()
-
-                    # uncomment the below section to draw the bounding box, drawing takes time and slow.
-                    # for image_idx, det_results in enumerate(prediction):
-                    #     for box_idx, bbox in enumerate(det_results):
-                    #         bbox = bbox.tolist()
-                    #         coord = bbox[:4]
-                    #         score = bbox[4]
-                    #         class_id = bbox[5]
-                    #         utils.plot_one_box(coord, self.org_image_list[image_idx],
-                    #             label="{}:{:.2f}".format(categories[int(class_id)], score))
-
-                except Exception as e:
-                    log.exception('Exception from Try is {}'.format(e))
-                    pass
+                # you can filter the prediction by class before nms. 
+                # ex: prediction = self.postprocess(['person']) 
```

Comment:
incorrect, index instead of class id.
Search for the index in the begining. just once

Reply:
fixed